apycula/clock_fuzzer.py

```diff
@@ -286,6 +300,7 @@ def branch_aliases(quads, clks):
             else:
                 src = "GBO1"
             for tap, branch_cols in taps.items():
+                add_rim(rows, branch_cols)
```

Comment:
So the else clause adds the last column to every single branch? That seems incorrect.

Reply:
you're right